.github/workflows/build_packages.yml

```diff
@@ -56,10 +60,12 @@ jobs:
       matrix:
         include:
           # Ubuntu packages.
+          - runs-on: ubuntu-24.04
+            platform: linux-x86_64
+            package: shark-ai
           - runs-on: ubuntu-24.04
             platform: linux-x86_64
             package: sharktank
-            python-version: cp311-cp311  # Ignored (generic wheel), set for workflow naming
```

Comment:
Keep the `python-version` + comments? Or adjust the `    name: "${{ matrix.package }} :: ${{ matrix.platform }} :: ${{ matrix.python-version }}"` above. One thing I've done in other workflows is have an explicit `name: `, so we'd have
```yml
          - name: shark-ai :: linux-x86_64
            runs-on: ubuntu-24.04
            platform: linux-x86_64
            package: shark-ai
...
          - name: shortfin :: linux-x86_64 :: 3.11
            runs-on: ubuntu-24.04
            platform: linux-x86_64
            package: shortfin
            python-version: cp311-cp311
```

Here's how this currently appears in the UI (https://github.com/nod-ai/SHARK-Platform/actions/runs/11855112109)
![image](https://github.com/user-attachments/assets/00e09832-cd47-49da-9636-5879aae5f1bc)

Reply:
It was just not used in the workflows for naming but we can keep it and add it there.

Reply:
It is used: https://github.com/nod-ai/SHARK-Platform/blob/8664abebcbde7542efaa85e339c0752bfad1f54c/.github/workflows/build_packages.yml#L50-L51

Reply:
My bad!

Reply:
(Re-)added the version + the comment.